node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -927,19 +929,27 @@ pub fn status() -> JsonRpcResultAsync {
         Err(e) => Err(internal_error_s(e)),
     });
 
-    let j = Future::join3(synchronized_fut, chain_beacon_fut, current_epoch_fut)
-        .map(|(synchronized, chain_beacon, current_epoch)| SyncStatus {
+    let j = Future::join4(
+        synchronized_fut,
+        chain_beacon_fut,
+        current_epoch_fut,
+        node_state_fut,
+    )
+    .map(
+        |(synchronized, chain_beacon, current_epoch, node_state)| SyncStatus {
             chain_beacon,
             current_epoch,
             synchronized,
```

Comment:
We should be able to remove the `synchronized_fut` if we do this:

```suggestion
            synchronized: node_state == Some(StateMachine::Synced),
```